client/src/cmdhfgallagher.c

```diff
@@ -865,89 +933,93 @@ static int CmdGallagherClone(const char *cmd) {
                   "   DES 8 bytes\n"
                   "   2TDEA or AES 16 bytes\n"
                   "   3TDEA 24 bytes\n"
-                  "AID id, default finds lowest available in range 0x2?81F4, where 0 <= ? <= 0xB.",
+                  "AID, default finds lowest available in range 0x??81F4, where ?? >= 0x20.",
                   "hf gallagher clone --rc 1 --fc 22 --cn 3333 --il 4 --sitekey 00112233445566778899aabbccddeeff"
                  );
 
     void *argtable[] = {
         arg_param_begin,
-        arg_int0("n",  "keynum",   "<dec>", "Key number [default = 0]"),
-        arg_str0("t",  "algo",     "<DES|2TDEA|3TDEA|AES>", "Crypt algo: DES, 2TDEA, 3TDEA, AES"),
-        arg_str0("k",  "key",      "<hex>", "Key for authentication to the PICC"),
+        arg_int0("n",   "keynum",  "<dec>", "PICC key number [default = 0]"),
+        arg_str0("t",   "algo",    "<DES|2TDEA|3TDEA|AES>", "PICC crypt algo: DES, 2TDEA, 3TDEA, AES"),
+        arg_str0("k",   "key",     "<hex>", "Key for authentication to the PICC to create applications (HEX 8(DES), 16(2TDEA or AES) or 24(3TDEA) bytes)"),
```

Comment:
we have limited screen space,  so long help-text for parameter is discouraged.  Instead write in the command help description if you need to make something clear for a param